frontend/projects/dipa-frontend/src/app/menus/gantt-menu/gantt-menu.component.html

```diff
@@ -2,4 +2,7 @@
   <div *ngFor="let timeline of timelineData">
     <button mat-menu-item [routerLink]="['/gantt/', timeline.id, 'project-checklist']">{{ timeline.name }}</button>
   </div>
+  <div *ngIf="timelineData.length === 0">
+    <button mat-menu-item disabled>keine Projekte</button>
+  </div>
```

Comment:
timelineData?.length ... sonst gibt's ein ERROR TypeError: Cannot read property 'length' of undefined